server/api/user/user.controller.js

```diff
@@ -522,7 +522,7 @@ exports.resetPassword = function(req, res){
 
               var sub = {
                 ":name": [user.name],
-                "[%address%]": [config.addr + "/login?token=" + user.passwordResetToken],
+                "[%address%]": ["https://rcos.io/login?token=" + user.passwordResetToken],
```

Comment:
Looks like this is actuallly just a config issue on the server. I'll dig into it but config.addr should be set already